.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -90,16 +90,17 @@ jobs:
           docker tag ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:latest
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:latest
 
-      - name: Update corresponding helm values file
-        uses: fjogeleit/yaml-update-action@main
+      - name: Update corresponding helm values file(with retry)
+        uses: Wandalen/wretry.action@master # Retries action on fail
         with:
-          valueFile: "k8s/netmanager/values-prod.yaml"
-          propertyPath: "image.tag"
-          value: ${{ needs.image-tag.outputs.build_id }}
-          updateFile: true
-          branch: deployment/argocd
-          message: "Update netmanger production image tag to ${{ needs.image-tag.outputs.build_id }}"
-          createPR: false
+          action: fjogeleit/yaml-update-action@main # Action to retry
+          with: |
+            valueFile: "k8s/netmanager/values-prod.yaml"
+            propertyPath: "image.tag"
+            value: ${{ needs.image-tag.outputs.build_id }}
+            branch: deployment/argocd
+            message: "Update netmanger production image tag to ${{ needs.image-tag.outputs.build_id }}"
+            createPR: false
```

Comment:
[update file with retry](https://github.com/airqo-platform/AirQo-frontend/pull/991/commits/4149af670794638b3ba16f265e72ad633a11fa7c)

Reply:
Hi @123MwanjeMike , the link in your comment leads to this PR. Intentional?

Reply:
Hi @Baalmart 
It leads to commit 4149af670794638b3ba16f265e72ad633a11fa7c whose commit message is "update file with retry"